packages/portfolio-contract/test/portfolio-open.test.ts

```diff
@@ -173,9 +174,9 @@ test('handle failure in executeEncodedTx', async t => {
   const localP = makeLocalAccount(orch, {});
   const actual = await openPortfolio(
     orch,
-    { zoeTools, makePortfolioKit, inertSubscriber },
+    { zoeTools, makePortfolioKit, inertSubscriber, contractAddresses },
     seat,
-    undefined,
+    { evmChain: 'Ethereum' },
```

Comment:
We have a `catch` for failures in `initRemoteEVMAccount` but I don't see a test here for it.